api/errors.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 type jsonError struct {
-	Code        string `json:"errorCode"`
+	Code        string `json:"code"`
```

Comment:
This has now introduced a breaking change, it would have been better to update the swagger spec to state errorCode instead of code as that means the swagger spec is also up to date for all the other endpoints that return an error.